contrib/colab/mnist-training.ipynb

```diff
@@ -201,32 +203,32 @@
         "    x = self.fc2(x)\n",
         "    return F.log_softmax(x, dim=1)\n",
         "\n",
+        "# Only instantiate model weights once in memory\n",
```

Comment:
Let's end comments like English sentences, with a dot.
Same in the matching comment in the other file.

Reply:
Done, now it's proper english 😄